src/pages/code-insights.tsx

```diff
@@ -271,14 +271,15 @@ const blogResourceItems = [
         href: '/blog/migrating-to-css-modules-with-codemods-and-code-insights',
     },
     {
-        title: 'Announcing Code Insights: analytics for engineering teams to understand and visualize their codebase over time',
-        description: 'Learn about why we built Code Insights from our CEO.',
-        type: 'Blog post',
+        title: '5 key traits of a code intelligence platform',
+        description:
+            'Sourcegraph is more than search. Learn how the code intelligence platform helps development teams quickly get unblocked, resolve issues faster, and gain insights to make better decisions.',
+        type: 'Guide',
         img: {
-            src: 'https://storage.googleapis.com/sourcegraph-assets/blog/code-insights-ga-blogs/announcement-header.png',
-            alt: 'Announcing code insights blog thumbnail',
+            src: '/backgrounds/dark-multi-grid.jpg',
```

Comment:
I was going to suggest that we change the src to be `string | ReactNode` to allow us to use our Background component here with the `darkMultiGrid` variant, but I want to improve the way we use backgrounds that now that we have Tailwind going. It's currently a WIP in my branch. I'd also like to move away from asset-heavy backgrounds and use more native code to generate interesting backgrounds. I'm currently working with @traceyljohnson to improve this.

The current Background component also assumes that all backgrounds are hero backgrounds so they're quite large assets to load for places where thumbnails would be more suitable. The only variants that have mobile and tablet versions are the `darkNebulous*` variants.

Maybe for now you can export a thumbnail version and place it in `/public/blog/thumbnails` or `/public/bg-images/thumbnails`. What do you think?

Reply:
Great points. And yes! Super interested to invest more in native code generated bg's 😄 

I was curious about our bg asset plan for some other scenarios like this in my draft PR right now. I updated the file path for this asset temporarily! Let me know if something else should change

Reply:
Sounds good! Path looks good for now!